embeddings/utils/onnx_exporter.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from transformers.models.auto import AutoFeatureExtractor, AutoProcessor, AutoTokenizer
+from transformers.onnx.convert import export, validate_model_outputs
+from transformers.onnx.features import FeaturesManager
+from transformers.onnx.utils import get_preprocessor
+
+from embeddings.model.lightning_module.huggingface_module import HuggingFaceLightningModule
+from embeddings.model.lightning_module.text_classification import TextClassificationModule
+from embeddings.utils.loggers import get_logger
+
+_logger = get_logger(__name__)
+
+
+@dataclass
+class OnnxExportConfiguration:
+    output: Path
+    model_name: str
+    model: HuggingFaceLightningModule
+
+
+class OnnxExporter:
```

Comment:
@riomus do we want to add LM optimizations such as quantization etc? we could reuse https://huggingface.co/transformers/v4.6.0/serialization.html this